test/recipes/80-test_ssl_old.t

```diff
@@ -577,44 +615,6 @@ sub testssl {
                'test auto DH meets security strength');
           }
 	}
-
-      SKIP: {
-            skip "TLSv1.1 is not supported by this OpenSSL build", 4
-                if $no_tls1_1;
-
-        SKIP: {
-            skip "skipping auto DHE PSK test at SECLEVEL 3", 1
-                if ($no_dh || $no_psk);
-
-            ok(run(test(['ssl_old_test', '-tls1_1', '-dhe4096', '-psk', '0102030405', '-cipher', '@SECLEVEL=3:DHE-PSK-AES256-CBC-SHA384'])),
-               'test auto DHE PSK meets security strength');
-          }
-
-        SKIP: {
-            skip "skipping auto ECDHE PSK test at SECLEVEL 3", 1
-                if ($no_ec || $no_psk);
-
-            ok(run(test(['ssl_old_test', '-tls1_1', '-no_dhe', '-psk', '0102030405', '-cipher', '@SECLEVEL=3:ECDHE-PSK-AES256-CBC-SHA384'])),
-               'test auto ECDHE PSK meets security strength');
-          }
-
-        SKIP: {
-            skip "skipping no RSA PSK at SECLEVEL 3 test", 1
-                if ($no_rsa || $no_psk);
-
-            ok(!run(test(['ssl_old_test', '-tls1_1', '-no_dhe', '-psk', '0102030405', '-cipher', '@SECLEVEL=3:RSA-PSK-AES256-CBC-SHA384'])),
-               'test auto RSA PSK does not meet security level 3 requirements (PFS)');
-          }
-
-        SKIP: {
-            skip "skipping no PSK at SECLEVEL 3 test", 1
-                if ($no_psk);
-
-            ok(!run(test(['ssl_old_test', '-tls1_1', '-no_dhe', '-psk', '0102030405', '-cipher', '@SECLEVEL=3:PSK-AES256-CBC-SHA384'])),
-               'test auto PSK does not meet security level 3 requirements (PFS)');
```

Comment:
Shouldn't these tests be converted to use TLSv1.2 instead? They don't seem TLSv1.1 specific.

Reply:
Okay.  Done.